packages/client/package.json

```diff
@@ -86,5 +86,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "node-fetch": "^2.6.1"
```

Comment:
move to devDep

Reply:
Shouldn't be needed here at all? This is only a concern of the test package AFAIK

Reply:
Moving it to `devDeps` but I can't remove it from this package because it's used inside `types.test.ts`:
 
![Screenshot 2023-02-04 at 08 31 25](https://user-images.githubusercontent.com/43268759/216755067-5a0405f2-4abb-4a40-a033-ddfdb7366d44.png)